pkg/machines/components/diskAdd.jsx

```diff
@@ -265,14 +265,15 @@ class PerformanceOptions extends React.Component {
 
 const CreateNewDisk = ({ idPrefix, onValueChanged, dialogValues, vmStoragePools, provider, vm }) => {
     const storagePool = vmStoragePools.find(pool => pool.name == dialogValues.storagePoolName);
+    const poolTypesNoSupportVolumeCreation = ['iscsi', 'iscsi-direct'];
```

Comment:
In addition to those, neither gluster nor mpath support volume creation.
Also, "poolTypesNotSupportingVolumeCreation" would sound more English.